blogweb/index.rst

```diff
@@ -40,6 +40,9 @@ the logos below to go to the respective sprint blog page.
 .. image:: _static/gsoc14.png
   :target: http://www.pointclouds.org/blog/gsoc14
   :class: foo
+.. image:: _static/simple_tree.png
+  :target: http://www.pointclouds.org/blog/gsoc14
```

Comment:
This is wrong, you want to point to `http://www.pointclouds.org/blogweb/simpletree`